src/downzip.js

```diff
@@ -9,9 +9,15 @@ const KEEPALIVE_INTERVAL_MS = 5000
 class DownZip {
     constructor(){
         this.worker = null
+    }
+
+    async register(options = {}) {

```

Comment:
Created an async register method. The downzip method was firing before the service worker was registered. Now we can await it.